qa-core/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "lib": ["es2020"],
+    "allowJs": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "incremental": true,
+    "types": ["node", "jest"],
+    "outDir": "dist",
+    "skipLibCheck": true,
+    "paths": {
+      "@budibase/types": ["../packages/types/src"],
+      "@budibase/backend-core": ["../packages/backend-core/src"],
+      "@budibase/backend-core/*": ["../packages/backend-core/*"]
```

Comment:
Do we need backend-core at all for the tests?

Reply:
Nope, I've removed it in my branch - this was just copied over from existing config.